### DISCUSSION TITLE
JUG Meeting the July  2021

### DISCUSSION THREAD

**Discussion by Llewellynvdm** [joomengine/Joomla-Component-Builder#761] [Announcements]
## You are all welcome

# Topic (possible)
- Discussion (why the corporate world is interested in JCB)

# Actions
- Help us [select the time](https://doodle.com/poll/6srg92um4eduqw24) of the next JUG meeting.
- Make more topics suggestions below.

**Comment by Llewellynvdm** [comment]
Topic Purposed:
- How to create a custom filter in JCB for the Admin List View.

**Comment by Llewellynvdm** [comment]
# MEETING TIME ANNOUNCEMENT!!

8th of July 2021 [see time](https://doodle.com/poll/6srg92um4eduqw24) all welcome at [room](https://meet.google.com/kxr-nfbv-dww).

> Please add this event to your [calendar](https://calendar.google.com/event?action=TEMPLATE&tmeid=M2RxcDQwZHM2aGVyZmxyNGwwZzFuOWlucjQgbGxld2VsbHluLnZhbi1kZXItbWVyd2VAY29tbXVuaXR5Lmpvb21sYS5vcmc&tmsrc=llewellyn.van-der-merwe%40community.joomla.org), and invite more JCB/Joomla developer... we really want to see you there!

**Reply by saman222** [reply]
hi @Llewellynvdm 
are you have JUG meeting in August?
we are awaiting for that.
also is it possible use Telegram for better jcb social connecting?

**Reply by Llewellynvdm** [reply]
Hey, sorry for not sending out a notice. But I am working night and day to move JCB to Joomla 4 this month and can't see a way to on-top of that still have a JUG. So apology for no JUG the month of August... we will try again in September.
